README.md

```diff
@@ -1,16 +1,118 @@
 # Digital Commerce Core Reference Template
 
-This is the base reference application for the apttus ecommerce product. Follow the below instructions to get started. See the [docs](https://cmoyle336.github.io/sdk-docs/overview.html) for more detailed instructions on interacting with the underlying SDK.
+This is the base reference application for the conga ecommerce product. Follow the below instructions to get started. See the [docs](https://apttus-ecom.github.io/May22/index.html) for more detailed instructions on interacting with the underlying SDK.
+
```

Comment:
@mpatidar-conga  SDK documentation link for december '21 isn't correct, it should be https://apttus-ecom.github.io/December21/index.html

Reply:
Changes added.